docs/source/guides/variables.rst

```diff
@@ -70,6 +76,46 @@ This is done simply by setting the :attr:`~Variable.grad` attribute of the outpu
 
 .. note::
 
-   Many functions taking :class:`Variable` object(s) are defined in the :mod:`~chainer.functions` module.
+   Many functions taking :class:`~chainer.Variable` object(s) are defined in the :mod:`chainer.functions` module.
    You can combine them to realize complicated functions with automatic backward computation.
 
+.. note::
+
+   Instead of using :func:`~chainer.Variable.backward`, you can also calculate gradients of any variables in a computational graph w.r.t. any other variables in the graph using the :func:`chainer.grad` function.
+
+
+Higher-Order Derivatives
+------------------------
+
+:class:`~chainer.Variable` also supports higher-order derivatives (a.k.a. double back-propagation).
+
+Let's see a simple example.
+First calculate the first-order derivative.
+Note that ``enable_double_backprop=True`` is specified to ``y.backward()``.
+
+.. doctest::
+
+    >>> x = chainer.Variable(np.array([[0, 2, 3], [4, 5, 6]], dtype=np.float32))
+    >>> y = x ** 3
+    >>> y.grad = np.ones((2, 3), dtype=np.float32)
+    >>> y.backward(enable_double_backprop=True)
+    >>> x.grad_var
+    variable([[  0.,  12.,  27.],
+              [ 48.,  75., 108.]])
+    >>> assert x.grad_var.array is x.grad
+    >>> assert (x.grad == (3 * x**2).array).all()
+
+:attr:`chainer.Variable.grad_var` is a :class:`~chainer.Variable` for :attr:`chainer.Variable.grad` (which is an :class:`~numpy.ndarray`).
+By specifying ``enable_double_backprop=True`` to ``backward()``, a computational graph for the backward calculation is recorded.
```

Comment:
`By passing` sound more natural (than `specifying ... to ...`). What do you think?